packages/google/Example/src/main/java/dev/hyo/martie/screens/uis/ProductCard.kt

```
@@ -111,21 +112,29 @@ fun ProductCard(
-                        "${firstOffer.priceCurrencyCode} ${String.format("%.2f", fullPrice)}",
-                        style = MaterialTheme.typography.bodyMedium,
-                        color = AppColors.textSecondary,
-                        textDecoration = androidx.compose.ui.text.style.TextDecoration.LineThrough
-                    )
+                    val fullPriceMicros = standardizedDiscount?.fullPriceMicrosAndroid?.toLongOrNull()
+                        ?: legacyOffer?.fullPriceMicros?.toLongOrNull()
+                    if (fullPriceMicros != null) {
+                        val fullPrice = fullPriceMicros.toDouble() / 1_000_000.0
+                        val currencyCode = standardizedDiscount?.currency ?: legacyOffer?.priceCurrencyCode ?: ""
+                        Text(
+                            "$currencyCode ${String.format("%.2f", fullPrice)}",
```

detekt (1.23.8)

[warning] String.format("%.2f", fullPrice) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)